engine/api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ProcessState.scala

```diff
@@ -44,7 +44,6 @@ object ProcessState {
       errors
     )
 
-  @JsonCodec case class StateStatusCodec(clazz: String, value: String)
```

Comment:
Is this class a part of public API? Should we change it?

Reply:
Nope, it was left by mistake. I know this from "source of primary change" :)